sydent/http/servlets/infoservlet.py

```diff
@@ -22,6 +22,7 @@
 import copy
 import yaml
 
+from netaddr import *
```

Comment:
Is this used anywhere?

Reply:
Ah, yes, that's a bug fix for previous thing which started using IPSet and IPNetwork but didn't import them.

Reply:
Addressed with an explicit import.